examples/cluster-dns/dns-frontend-pod.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   containers:
     - name: dns-frontend
-      image: ddysher/dns-frontend
+      image: gcr.io/google_samples/dns-frontend:v1
```

Comment:
change this to `gcr.io/google_containers/example-dns-frontend:v1` this image now exists.